tf2onnx/onnx_opset/math.py

```diff
@@ -566,10 +566,23 @@ def version_12(cls, ctx, node, **kwargs):
         del node.attr["N"]
 
 
-@tf_op("MatrixInverse", onnx_op="Inverse")
-class Inverse:
```

Comment:
BTW, are we dropping MatrixInverse operator? Seems to be useful (or was this a duplicate implementation?)

Reply:
the problem with MatrixInverse is that it depends on the Inverse operator that was taken out of opset-12 and if it is not in onnx it can't be used in onnx_opset/*, aka doing node.domain is MS_DOMAIN can't be in onnx_opset/*.
But that same code is also here:
https://github.com/onnx/tensorflow-onnx/blob/master/tf2onnx/custom_opsets/ms.py#L107
so this works as long you enable ms-domain.

Reply:
Ok, that makes sense. It looks like Inverse was dropped after 2 other ops Onnx ops were dropped 

Yup, agreed that MS_DOMAIN cannot be used in any general ops.